frontend/src/student-ui/student-ui.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { useMemo, useState } from 'react';
+import Button from '@openedx/paragon/dist/Button';
+import Icon from '@openedx/paragon/dist/Icon';
+import CollapsibleAdvanced from '@openedx/paragon/dist/Collapsible/CollapsibleAdvanced';
+import CollapsibleTrigger from '@openedx/paragon/dist/Collapsible/CollapsibleTrigger';
+import CollapsibleBody from '@openedx/paragon/dist/Collapsible/CollapsibleBody';
+import CollapsibleVisible from '@openedx/paragon/dist/Collapsible/CollapsibleVisible';
+import KeyboardArrowDown from '@openedx/paragon/icons/es5/KeyboardArrowDown';
+import KeyboardArrowUp from '@openedx/paragon/icons/es5/KeyboardArrowUp';
```

Comment:
Question: Why do you have these deep imports instead of just importing from `@openedx/paragon` ? Doesn't vite have really good tree-shaking that would have the same effect?

Reply:
It's been a while so I'm not entirely certain, but I was continuously monitoring the bundle size and I think doing it this way did reduce it. 

For the icons, as far as  I can remember, somewhere in the chain there is an 'export * from' and that was breaking the build.

Reply:
Hmm, interesting. If you notice it again, please take the time to open an upstream issue about it or even a PR.

Reply:
I have seen huge Paragon bundles with webpack, but when using Vite I generally found that it only included what we actually used (i.e. that tree-shaking worked).